### ISSUE TITLE
Flush not work as expected.

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#2636]
**Describe the bug**
Flush is too slowly.
```
11:33:38  2020-06-20:03:33:38,280 INFO     [runner.py:171] Start id: 9999800, end id: 9999900
11:33:38  2020-06-20:03:33:38,284 INFO     [client.py:34] Milvus insert run in 0.0s
11:33:38  2020-06-20:03:33:38,285 DEBUG    [runner.py:175] 0
11:33:38  2020-06-20:03:33:38,286 INFO     [runner.py:171] Start id: 9999900, end id: 10000000
11:33:38  2020-06-20:03:33:38,289 INFO     [client.py:34] Milvus insert run in 0.0s
11:33:38  2020-06-20:03:33:38,291 DEBUG    [runner.py:175] 0
11:33:38  2020-06-20:03:33:38,291 INFO     [k8s_runner.py:111] {'total_time': 589.29, 'qps': 16969.51, 'ni_time': 0.01}
```
Flush last 40mins and not finished, data are not created under `db` dir.
```
➜  sift_10m_128_128_l2_flush du -sh *
165K	db
176M	logs
4.9G	wal
```
After 50min passed, parts of data created.
```
➜  sift_10m_128_128_l2_flush du -sh *
3.3G	db
178M	logs
1.8G	wal
```

error message not found, the following debug log shows `[insert][0]Insert into segment` continually.
```
2020-06-20 12:10:01,234 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
2020-06-20 12:10:01,278 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
2020-06-20 12:10:01,321 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
2020-06-20 12:10:01,365 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
2020-06-20 12:10:01,409 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
2020-06-20 12:10:01,454 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
2020-06-20 12:10:01,499 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
2020-06-20 12:10:01,543 | DEBUG | default | [ENGINE] [insert][0]Insert into segment
```

**Steps/Code to reproduce behavior**
1. create collection, set auto_flush_interval=3000
2. insert 100 vectors/per request, insert sift-10m totally.
3. start flush

**Expected behavior**

**Environment details**
0.8.1
pymilvus-test: 0.2.62

**Screenshots**

**Additional context**

**Comment by JinHai-CN** [comment]
How about turn off WAL?

**Comment by del-zhenwu** [comment]
If disable WAL, and set auto_flush_interval=30000 or set auto_flush_interval=0(disable autoflush), disable autoflush not work.
```
19:49:52  2020-06-20:11:49:51,968 INFO     [runner.py:171] Start id: 8380000, end id: 8385000
19:49:52  2020-06-20:11:49:52,140 INFO     [client.py:34] Milvus insert run in 0.17s
19:49:52  2020-06-20:11:49:52,143 DEBUG    [runner.py:175] 0
19:49:52  2020-06-20:11:49:52,170 INFO     [runner.py:171] Start id: 8385000, end id: 8390000
19:49:52  2020-06-20:11:49:52,336 INFO     [client.py:34] Milvus insert run in 0.17s
19:49:52  2020-06-20:11:49:52,338 DEBUG    [runner.py:175] 0
19:49:52  2020-06-20:11:49:52,352 INFO     [runner.py:171] Start id: 8390000, end id: 8395000
19:50:10  2020-06-20:11:50:09,54 INFO     [client.py:34] Milvus insert run in 16.7s
19:50:10  2020-06-20:11:50:09,56 DEBUG    [runner.py:175] 8390000
19:50:10  2020-06-20:11:50:09,76 INFO     [runner.py:171] Start id: 8395000, end id: 8400000
19:50:10  2020-06-20:11:50:09,146 INFO     [client.py:34] Milvus insert run in 0.07s
19:50:10  2020-06-20:11:50:09,148 DEBUG    [runner.py:175] 8390000
```

**Comment by JinHai-CN** [comment]
This case is to write NFS or local disk?

**Comment by del-zhenwu** [comment]
> This case is to write NFS or local disk?

NFS. data/log path: `/test/milvus/db_data_080/sift_10m_128_128_l2_flush_debug`

**Comment by yhmo** [comment]
Use c++ sdk example to compare insert performance of 0.8.1 and 0.10.1:
0.8.1:
```
Insert 100000 entities No.0 totally cost: 785 ms
Insert 100000 entities No.1 totally cost: 647 ms
Insert 100000 entities No.2 totally cost: 568 ms
Insert 100000 entities No.3 totally cost: 588 ms
Insert 100000 entities No.4 totally cost: 1876 ms
Flush begin...
Flush totally cost: 4042 ms
```

0.10.1:
```
Insert 100000 entities No.0 totally cost: 870 ms
Insert 100000 entities No.1 totally cost: 665 ms
Insert 100000 entities No.2 totally cost: 527 ms
Insert 100000 entities No.3 totally cost: 532 ms
Insert 100000 entities No.4 totally cost: 567 ms
Flush begin...
Flush totally cost: 1086 ms
```

**Comment by shengjun1985** [comment]
Fixed in 0.10.1
